contracts/tokenization/StakedTokenFactory.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "./StakedToken.sol";
+
+contract StakedTokenFactory {
+    function createStakedToken(address stakeManagerAddress) external returns (address) {
+        StakedToken sToken = new StakedToken(stakeManagerAddress);
```

Comment:
create IstakedToken which has only mint() and burn()

Reply:
done